experimenter/experimenter/features/configs/apps.yaml

```
@@ -5,16 +5,20 @@
 fenix:
   repo: @mozilla-mobile/firefox-android
   fml_path: fenix/app/nimbus.fml.yaml
-  branch_mapping: "v{version}"
+  major_release_branch: "releases_v{major}"
+  minor_release_tag: "fenix-v{major}.{minor}"
 ios:
```

Was just chatting with @jhugman and realized these should be matching these [`app_names`](https://github.com/mozilla/experimenter/blob/36b973d502f765b698f7b16439fcf875492db10e/experimenter/experimenter/experiments/constants.py#L93) (not the slugs)
```suggestion
firefox_ios:
```